src/components/UserNicknamesPanel/UserNicknamesPanel.js

```diff
@@ -57,6 +57,19 @@ function UserNicknamesPanel () {
     return undefined
   }, [dispatch, nicknames, user])
 
+  const handleSetDisplayNickname = useCallback(async (event) => {
+    const nickname = nicknames.find((nick) => {
+      return nick.id === event.target.name
+    })
+
+    const response = await dispatch(setDisplayNickname(nickname.id, nickname.attributes.nick))
+
+    if (!isError(response)) {
```

Comment:
We could file this as a future enhancement instead, but I think we could easily add error handling  here. We already show errors for deletion, so we can re-use the `error` `useState` for that without changing the UI.

If we do change this now, could we also add `setError(null)` at the top of both action handlers? That way we reset the error state making it clear that if the error repeats, it's the error for the latest user interaction, and not a previous one.